recipes-bsp/bootfiles/rpi-config_git.bb

```diff
@@ -206,7 +208,7 @@ do_deploy() {
     # ENABLE CAN
     elif [ "${ENABLE_CAN}" = "1" ]; then
```

Comment:
I reckon this should now be also used in `ENABLE_DUAL_CAN`. @colin-pm , what do you think?

Reply:
I think for consistency the variable might as well get used for single and dual CAN interfaces so this wont have to be revisited when a dual CAN board shows up on the market with a different oscillator.

Reply:
Agreed. Let's do that and move the variable description in docs after `ENABLE_DUAL_CAN`.

Reply:
Alright, I added the configruation also for dual CAN and combined the documentation together.